### PR TITLE
Changed math to mkldnn_math_node

### PR DESCRIPTION
**Pull request by egorshul**

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+    initializers[op->get_type_info()](op, *this);
+    typeOp = MKLDNNPlugin::one_of(op->get_type_info(), ngraph::op::v0::HardSigmoid::type_info, ngraph::op::v0::Selu::type_info) ? 3 : 1;
+}
+
+void MKLDNNMathNode::initSupportedPrimitiveDescriptors() {
+    if (!supportedPrimitiveDescriptors.empty())
+        return;
+
+    if (typeOp == 3) {
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp:55]: You can generalize the logic by creating separate std::vector<DataConfigurator> with required number of  {TensorDescCreatorTypes::ncsp, Precision::FP32} elements. Number of inputs: getParentEdges().size()

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+#include <cmath>
+#include <vector>
+#include <string>
+#include <mkldnn_types.h>
+#include "ie_parallel.hpp"
+#include <mkldnn_selective_build.h>
+#include "mkldnn_math_node.h"
+#include <nodes/common/tensor_desc_creator.h>
+#include <ngraph/opsets/opset2.hpp>
+#include "utils/general_utils.h"
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp:14]: Please remove unused includes

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp (hunk 1 above)
Comment (egorshul) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp]: Fixed in 8c979d2088b5538bf6caf32c81e30a5b9dd485d1. Thank you!

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp (hunk 2 above)
Comment (egorshul) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp:12]: Fixed in 8c979d2088b5538bf6caf32c81e30a5b9dd485d1. Thank you!

**Comment by dmitry-gorokhov** [comment]
@egorshul please resolve conflicts

**Comment by egorshul** [comment]
@dmitry-gorokhov Resolved, thanks

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**